composer.json

```diff
@@ -18,9 +18,9 @@
         "webmozart/assert": "^1.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.5",
+        "phpunit/phpunit": "^7.5",
         "symfony/phpunit-bridge": "^3.4",
-        "symplify/easy-coding-standard": "^7.0.2"
+        "symplify/easy-coding-standard": "^8.2.3"
```

Comment:
It will have the dependency problem.

The `symplify/easy-coding-standard:^8.2.3` version should require `php-7.2` version at least.

And this will be failed on `php-7.1` version.

Reply:
I'm thinking about dropping PHP 7.1, raising the minimum to 7.2. Keeping in sync with Symfony.

Reply:
I think it should be good and I just notice that we should create new release version for this change.

Reply:
Yes, let's bump it to v3, when this is working! :-)